cmd/service_bootstrap_desktop.go

```diff
@@ -114,19 +115,31 @@ func (di *Dependencies) bootstrapServiceOpenvpn(nodeOptions node.Options) {
 		currentLocation := market.Location{Country: locationInfo.Country}
 		transportOptions := serviceOptions.(openvpn_service.Options)
 
-		mapPort := func() func() {
+		mapPort := func(port int) func() {
 			return mapping.GetPortMappingFunc(
 				locationInfo.PubIP,
 				locationInfo.OutIP,
 				transportOptions.Protocol,
-				transportOptions.Port,
```

Comment:
We cannot remove transportOptions altogether. User should be left with an option to configure their own, static port. Default value should be removed, or set to '0' and only if it is '0' we should activate pool logic.

Reply:
Fixed